### DISCUSSION TITLE
Any success debugging React on Quest Browser with React Dev Tools?

### DISCUSSION THREAD

**Discussion by mz8i** [pmndrs/xr#233] [Q&A] [answered]
This is only tangentially related to the project, but I figured out if anyone would have the same problem as me then it would be users of this repo, given it's on the intersection of React and XR.

Anyone found a way to debug a react-xr application running on Quest Browser, so that you could use the React Dev Tools plugin to debug components etc?
Just connecting from a Chrome on desktop and debugging the Quest Browser to see console logs etc is pretty straightforward, but I haven't been able to figure out if it's possible at all using something like React Dev Tools.
Normally, when debugging React apps that are running locally, the React Dev Tools extension injects code into the site that then allows the dev tools to connect and inspect that component hierarchy. But on Quest, there's no extension that could inject the code...

Anyone had a similar goal? Any luck or workarounds?

**Comment by saitonakamura** [comment, marked as answer]
You can you https://beta.reactjs.org/learn/react-developer-tools#safari-and-other-browsers react-devtools as a package
